pxr/usdImaging/lib/usdviewq/appController.py

```diff
@@ -3177,27 +3177,30 @@ def setFrame(self, frameIndex, forceUpdate=False):
             str(round(self._dataModel.currentFrame.GetValue(), ndigits=2)))
 
         if (self._dataModel.currentFrame != frameAtStart) or forceUpdate:
-            # do not update HUD/BBOX if scrubbing or playing
-            updateUI = forceUpdate or not (self._dataModel.playing or
-                                          self._ui.frameSlider.isSliderDown())
-            self._updateOnFrameChange(updateUI)
+            self._updateOnFrameChange()
 
-    def _updateOnFrameChange(self, refreshUI = True):
-        """Called when the frame changed, updates the renderer and such"""
-
-        if refreshUI: # slow stuff that we do only when not playing
-            # topology might have changed, recalculate
-            self._updateHUDGeomCounts()
-            self._updatePropertyView()
-            self._refreshAttributeValue()
+    def _updateOnFrameChangeFinish(self):
```

Comment:
Given that this method is responsible basically for updating the entire GUI except the viewport, I think maybe a name like _updateGUIForFrameChange() might give better guidance?  Then _updateOnFrameChange() can discuss that when we're in playback, we don't update the GUI except for the viewport and transport controls?

Reply:
Agreed - updated!